clang/include/clang/AST/CXXRecordDeclDefinitionBits.def

```diff
@@ -189,6 +189,11 @@ FIELD(DeclaredNonTrivialSpecialMembers, 6, MERGE_OR)
 /// SMF_MoveConstructor, and SMF_Destructor are meaningful here.
 FIELD(DeclaredNonTrivialSpecialMembersForCall, 6, MERGE_OR)
 
+/// True when this class's bases and fields are all trivially relocatable
+/// or references, and the class itself has no user-provided special
+/// member functions.
+FIELD(IsNaturallyTriviallyRelocatable, 1, NO_MERGE)
```

Comment:
I know you are trying to distinguish from the abi_tag but naturally is not saying much.

Either `IsTriviallyRelocatable` or, `IsImplicitlyTriviallyRelocatable`